packages/smooth_app/lib/pages/personalized_ranking_page.dart

```diff
@@ -111,7 +113,7 @@ class _PersonalizedRankingPageState extends State<PersonalizedRankingPage> {
                 (final int index) => Tab(
                   child: Text(
                     titles[index],
-                    style: TextStyle(color: colors[index]),
+                    style: index==0?TextStyle(color:themeData.hintColor ):TextStyle(color: colors[index]),
```

Comment:
Could you reformat your code please?

Reply:
hi @g123k 
I have now re-format the code.